apps/omni/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "omni",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "generate": "graphql-codegen --config codegen.yaml",
+    "compile": "npm run generate && tsc",
+    "dev": "nodemon -e ts --exec \\\"(tsc || exit 1) && node dist/index.js\\\"",
```

Comment:
why the `\\\` and not `\`

Reply:
Because the actual command needs to be `\"something\"` because I am escaping for bash, but I also need to escape for JSON so there is double escaping going on

Reply:
Annoying but it's the way I managed to get it working without ts-node because ts-node has some issues with certain tsconfigs